exercises/robot-simulator/src/main/java/GridPosition.java

```diff
@@ -0,0 +1,16 @@
+public final class GridPosition {
+
+    final int x;
+
+    final int y;
+
+    GridPosition(final int x, final int y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    boolean equals(final GridPosition gridPosition) {
```

Comment:
I think your proposal is a good balance, and will take a stab at it locally with the expectation that we might iterate a couple of times to capture all that needs to be said!

Related-ish: are reference solutions ever exposed to users? I ask in light of [this comment](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#reference-solution) (my emphasis):

> The reference solution is named something with example or Example in the path.
> 
> **The solution does not need to be particularly great code, it is only used to verify that the exercise is coherent.**

I think that exposing users to exemplary reference solutions should be the norm; just wondering where Exercism as a whole stands on that front?